src/components/AboutUsCard.js

```diff
@@ -52,8 +53,9 @@ export default function AboutUsCard({developer}) {
     }));
 
     return (
-        <div className="about-us-card">
-        <Card sx={{ display: 'flex' , background: 'darkgray', width: 'fit-content' }}>
+        <ThemeProvider theme={theme}>
```

Comment:
Couldn't we just wrap the `App` component with `ThemeProvider` so we only have to add it once?

Reply:
You're right, I just pushed the new changes